### DISCUSSION TITLE
Integration website - wopiserver - collabora

### DISCUSSION THREAD

**Discussion by jmlm74** [cs3org/wopiserver#42] [Q&A] [answered]
Hi,
I'd like to implement wopiserver and colabora with a website.
Could you explain me what should be the best way to do.

Collabora is installed and is working fine on local
I've just installed the wopi server.
 - Starting the server is ok
 - Executing wopiopen.py is OK --> give me the wopi url and the access token
 - Building manually the url and open document is OK.
 
 But now I don't know how to go further. 
 --> Open an existing document or create a new one directly from my website --> get the url or build it automatically.

  Any help should be appreciated.

 Thanks in advance

**Comment by glpatcern** [comment, marked as answer]
Hi there,

First apologies for the very late reply, for some reasons I missed the notification of this discussion and I found it now by pure chance.

Now, not sure if you already sorted out your issue, but as a matter of fact the concept here is to interconnect Collabora (or other WOPI-compatible editors) to a sync&share cloud storage system, not just to a website. True that this wopiserver supports local storage, but it was implemented for testing purposes only and I would not consider it good for any production activity beyond testing, apart that you would need to sort out the backend logic to "emulate" what a sync&share web service does behind the scene.

Please let us know if you're interested in adopting the solutions we are developing and integrating instead. More details about your use case would be useful.

Cheers,
Giuseppe
(Contributor in the CS3Org and main author of the CS3 wopiserver)
